lib/bolt/outputter/human.rb

```diff
@@ -180,6 +184,18 @@ def print_step_finish(description:, result:, duration:, **_kwargs)
         @stream.puts(colorize(:green, message))
       end
 
+      def print_container_start(image:, **_kwargs)
+        @stream.puts(colorize(:green, "Starting: run container #{image}"))
+      end
+
+      def print_container_finish(image:, success:, **_kwargs)
+        if success
+          @stream.puts(colorize(:green, "Finished: run container '#{image}' succeeded."))
+        else
+          @stream.puts(colorize(:red, "Finished: run container '#{image}' failed."))
+        end
+      end
```

Comment:
Putting this here since it's related to output. It'd be nice if `#print_plan_result` handled `ContainerResult` specifically. Right now it just pretty prints the class as JSON.